docs/source/admin/index.rst

```diff
@@ -23,38 +23,185 @@ Kubernetes deployment
-       | For a quick test, run ``python -m http.server`` inside the ``html`` directory.
-    4. | Try fetching ``http://<your-server>/api/v1/``.
-       | In the above test example: ``http://0.0.0.0:8000/api/v1``.
-    5. Make sure that the registry server is accessible from the AiiDAlab server.
+.. panels::
+   :container: container-lg pb-3
+   :column: col-lg-12 p-2
 
-Use the new registry:
+   .. _admin-guide:maintain-app-registry:create:
 
-    1. | When creating the AiiDAlab docker container, pass the URL to the registry server to the container in the ``AIIDALAB_REGISTRY`` environment variable (e.g. using `docker-compose <https://docs.docker.com/compose/reference/>`_).
-       | When running a test registry on the docker host, pass the following flags to ``docker run``:
+   **Create the new registry**
 
-        * ``--add-host=host.docker.internal:host-gateway`` (only required on Linux, not MacOS)
-        * ``-e AIIDALAB_REGISTRY=http://host.docker.internal:8000/api/v1``
+   To create a registry, first make sure to install the ``aiidalab`` package on the machine that you want to *build* the registry on.
 
-To verify that the new registry is being used, open a terminal in an AiiDAlab container and run:
+   .. code-block:: console
 
-.. code-block:: bash
+        $ pip install aiidalab
+
+   .. note::
+
+        For testing, you could build the registry on a running AiiDAlab instance, in this case the ``aiidalab`` package is already installed.
+
+   Next, create a directory for your registry repository, e.g., with:
+
+   .. code-block:: console
+
+      ~$ mkdir my-aiidalab-registry
+      ~$ cd my-aiidalab-registry/
+      ~/my-aiidalab-registry$
+
+   Then create two files: a :file:`apps.yaml` and a :file:`categories.yaml` file.
+   The first one contains our index of registered applications and the second one the available categories for apps in this registry.
+
+   The definition for entries in the ``apps.yaml`` file are described in detail in :ref:`the documentation on app registration <develop-apps:publish-app:register>`.
+   Example:
+
+   .. code-block:: yaml
+       :caption: apps.yaml
+
+        hello-world:
+          releases:
+            - "git+https://github.com/aiidalab/aiidalab-hello-world@master:"
+
+   The ``categories.yaml`` file is a dictionary where, the key of each entry is the category's id and the value consist of a ``title`` and a ``description`` field.
+   Example:
+
+   .. code-block:: yaml
+      :caption: categories.yaml
+
+       classical:
+         description: Apps for performing calculations based on classical/empirical force
+           fields.
+         title: Classical
+       quantum:
+         description: Apps for performing quantum-mechanical calculations.
+         title: Quantum
+
+   .. tip::
+
+        Instead of creating a new repository from scratch, you can also fork the official `AiiDAlab registry repository <https://github.com/aiidalab/aiidalab-registry>`_ and adjust it for your needs.
```

Comment:
I think something like this needs to be mentioned in the beginning; otherwise people just start following your instructions